### ISSUE TITLE
Aria label key words of virtual backgrounds are not translatable

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/spreed#9610]
> The words are still not translated, but at least no TODO aria labels anymore. Will have to think about a way to translate them.
Will raise a follow up issue for that.

_Originally posted by @nickvergessen in https://github.com/nextcloud/spreed/pull/9609#discussion_r1204347510_

**Comment by ShGKme** [comment]
@marcoambrosini @Antreesy what do you think?

**Comment by Antreesy** [comment]
> What about making a constant with filename-to-description map?

As far as I know, It will be impossible for translators to translate differently `Office` and `Select virtual {{insert Office here}} background`.
For Russian and German langs, for example, inserted part will be in the end and in different case (Fall :de:  / падеж :ru: )

But constant with complete sentence should work fine, and doesn't require a lot of efforts from translators:
` '1_office': t('spreed', 'Select virtual Office background'),`

**Comment by ShGKme** [comment]
> Currently, background image presets are hardcoded in the source.

I forgot that they are actually got from 
```js
getCapabilities()?.spreed?.config?.call?.['predefined-backgrounds']
```

Which is defined on runtime according to the `img/backgrounds/` content

https://github.com/nextcloud/spreed/blob/b8eb2658d31f75a584f4124afde798232835b997/lib/Capabilities.php#L176-L205

End-user is not supposed to edit the app's source, but it is possible.

**Comment by Antreesy** [comment]
> End-user is not supposed to edit the app's source, but it is possible.

The new constant is not exposed, so shouldn't be a big problem, i guess.
Or, we could send translations and filepath together, via capabilities, in PHP-code (it supports t('spreed'), if I'm not wrong)

**Comment by ShGKme** [comment]
> The new constant is not exposed, so shouldn't be a big problem, i guess.
> We could send translations and filepath together, via capabilities, in PHP-code (it supports t('spreed'), if I'm not wrong). But it's going to be hardcoded in another place then.

It is changeable not by constant editing in the PHP/JS source code, but by updating files in the `img/backgrounds/` folder. Translations on the server side cannot help here either.
